ContentProtection/BasicWidevine/Program.cs

```diff
@@ -252,12 +251,25 @@ private static async Task RunAsync(ConfigWrapper config)
         /// </summary>
         /// <param name="config">The param is of type ConfigWrapper. This class reads values from local configuration file.</param>
         /// <returns></returns>
+        // <GetCredentialsAsync>
         private static async Task<ServiceClientCredentials> GetCredentialsAsync(ConfigWrapper config)
         {
-            // Use ApplicationTokenProvider.LoginSilentAsync to get a token using a service principal with symmetric key
-            ClientCredential clientCredential = new ClientCredential(config.AadClientId, config.AadSecret);
-            return await ApplicationTokenProvider.LoginSilentAsync(config.AadTenantId, clientCredential, ActiveDirectoryServiceSettings.Azure);
+            // Use ConfidentialClientApplicationBuilder.AcquireTokenForClient to get a token using a service principal with symmetric key
+
+            var scopes = new[] { config.ArmAadAudience + "/.default" };
```

Comment:
Should you add a comment here about what and why the "/.default" string is being used? 
Should we put that string in a constant?

Reply:
/.default works but we should check why. I don't know if this is the best
https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-v1-app-scopes

Reply:
Maybe wait for Jon Payne to review this?